test/potluck_test.rb

```diff
@@ -0,0 +1,137 @@
+require "minitest/autorun"
+require "minitest/pride"
+require "./lib/dish.rb"
+require "./lib/potluck.rb"
+
+class PotluckTest < Minitest::Test
+  def test_it_exists
+    potluck = Potluck.new("7-13-18")
+    assert_instance_of Potluck, potluck
+  end
+
+  def test_it_has_date
+    potluck = Potluck.new("7-13-18")
+    assert_equal "7-13-18", potluck.date
+  end
+
+  def test_it_has_no_dishes
+    potluck = Potluck.new("7-13-18")
+    assert_equal [], potluck.dishes
+  end
+
+  def test_it_adds_dishes
+    potluck = Potluck.new("7-13-18")
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+    potluck.add_dish(couscous_salad)
+    assert_equal [couscous_salad], potluck.dishes
+    potluck.add_dish(cocktail_meatballs)
+    assert_equal [couscous_salad, cocktail_meatballs], potluck.dishes
+  end
+
+  def test_it_has_correct_number_dishes
+    potluck = Potluck.new("7-13-18")
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+    potluck.add_dish(couscous_salad)
+    assert_equal 1, potluck.dishes.length
+    potluck.add_dish(cocktail_meatballs)
+    assert_equal 2, potluck.dishes.length
+  end
+
+  def test_it_gets_all_from_category
+    potluck = Potluck.new("7-13-18")
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    summer_pizza = Dish.new("Summer Pizza", :appetizer)
+    roast_pork = Dish.new("Roast Pork", :entre)
+    cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+    candy_salad = Dish.new("Candy Salad", :dessert)
+    potluck.add_dish(couscous_salad)
+    potluck.add_dish(summer_pizza)
+    potluck.add_dish(roast_pork)
+    potluck.add_dish(cocktail_meatballs)
+    potluck.add_dish(candy_salad)
+    assert_equal [couscous_salad, summer_pizza], potluck.get_all_from_category(:appetizer)
+    assert_equal [roast_pork, cocktail_meatballs], potluck.get_all_from_category(:entre)
+    assert_equal [candy_salad], potluck.get_all_from_category(:dessert)
+  end
+
+  def test_it_returns_first_item_of_category
```

Comment:
You don't need to test for getting the first thing. .first is an array method so it's not this tests job to test that.